src/main/java/com/ecwid/consul/v1/agent/model/Self.java

```diff
@@ -20,57 +20,106 @@
 
 	public static class Config {
 
-		@SerializedName("Bootstrap")
-		private boolean bootstrap;
+		@SerializedName("Datacenter")
+		private String datacenter;
+
+		@SerializedName("NodeName")
+		private String nodeName;
+
+		@SerializedName("Revision")
+		private String revision;
 
 		@SerializedName("Server")
 		private boolean server;
 
-		@SerializedName("Datacenter")
-		private String datacenter;
+		@SerializedName("Version")
+		private String version;
+
+		public String getDatacenter() {
+			return datacenter;
+		}
+
+		public void setDatacenter(String datacenter) {
+			this.datacenter = datacenter;
+		}
+
+		public String getNodeName() {
+			return nodeName;
+		}
+
+		public void setNodeName(String nodeName) {
+			this.nodeName = nodeName;
+		}
+
+		public String getRevision() {
+			return revision;
+		}
+
+		public void setRevision(String revision) {
+			this.revision = revision;
+		}
+
+		public boolean isServer() {
+			return server;
+		}
+
+		public void setServer(boolean server) {
+			this.server = server;
+		}
+
+		public String getVersion() {
+			return version;
+		}
+
+		public void setVersion(String version) {
+			this.version = version;
+		}
+
+		@Override
+		public String toString() {
+			return "DebugConfig{" +
+							"datacenter='" + datacenter + "'" +
+							", nodeName='" + nodeName + "'" +
+							", revision='" + revision + '\'' +
+							", server=" + server +
+							", version='" + version + '\'' +
+							'}';
+		}
+
+	}
+
+	public static class DebugConfig {
```

Comment:
like @slackpad suggested, we recommend not parsing DebugConfig. Its really an internal representation and we will add or remove fields in the future.